docs/daikon/daikon.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Daikon REST API
-nav_order: 3
+nav_order: 2
```

Comment:
Not completely sure why the `nav_order`s changed in these files but hopefully you inspected the page locally and saw it was alright.

Reply:
Oh, forgot to describe this in the commit message. In this one, there were only two to begin with (home and daikon) so I updated this to two since contributing would become 3. And as for the other one, two of them were labeled nav_order: 3, so I just updated one to 4 (which was already showing as the 4th one on the website)